.github/workflows/reusable-build-provider.yaml

```diff
@@ -140,7 +140,8 @@ jobs:
         env:
           COSIGN_YES: true
         run: |
-          IMAGE=$(FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=${{ inputs.arch }} VARIANT=${{ inputs.variant}} REGISTRY_AND_ORG="quay.io/kairos" RELEASE=master kairos-agent versioneer container-artifact-name)
+          K3S_VERSION=$(sudo luet --config framework-profile.yaml search -o json k8s/k3s  | jq '.packages | map(.version) | unique | last' | tr -d '"')
```

Comment:
Hopefully, no new k3s version appears between this call and the other one in the previous step (build) :D

Reply:
and I just relized this will be problematic because we will not overwrite older k3s versions .... maybe it's better not to fix, wdyt @jimmykarily ?

Reply:
I'll go with it and we can clean manually worst case. I followed up with quay expiration dates but I don't think they will solve it :/